Description: Use the system clang instead of the one that is (not really)
 inside the source tree.
Author: Chad Miller <chad.miller@canonical.com>

Index: dev.vivid/build/common.gypi
===================================================================
--- dev.vivid.orig/build/common.gypi	2015-04-22 14:43:01.395664661 -0400
+++ dev.vivid/build/common.gypi	2015-04-22 14:43:01.373664971 -0400
@@ -2363,8 +2363,8 @@
       }],
 
       ['host_clang==1', {
-        'host_cc': '<(make_clang_dir)/bin/clang',
-        'host_cxx': '<(make_clang_dir)/bin/clang++',
+        'host_cc': 'clang',
+        'host_cxx': 'clang++',
       }, {
         'host_cc': '<!(which gcc)',
         'host_cxx': '<!(which g++)',
Index: dev.vivid/native_client/build/common.gypi
===================================================================
--- dev.vivid.orig/native_client/build/common.gypi	2015-04-22 14:43:01.395664661 -0400
+++ dev.vivid/native_client/build/common.gypi	2015-04-22 14:43:01.383664831 -0400
@@ -500,8 +500,8 @@
             ['chromium_mac_pch', {'GCC_PRECOMPILE_PREFIX_HEADER': 'YES'},
                                  {'GCC_PRECOMPILE_PREFIX_HEADER': 'NO'}],
             ['clang==1', {
-              'CC': '$(SOURCE_ROOT)/<(clang_dir)/clang',
-              'LDPLUSPLUS': '$(SOURCE_ROOT)/<(clang_dir)/clang++',
+              'CC': 'clang',
+              'LDPLUSPLUS': 'clang++',
               'GCC_VERSION': 'com.apple.compilers.llvm.clang.1_0',
               'WARNING_CFLAGS': [
                 '-Wheader-hygiene',
